sympy/polys/tests/test_numberfields.py

```diff
@@ -545,6 +545,7 @@ def test_AlgebraicNumber():
     assert a.is_aliased is True
 
     assert AlgebraicNumber(sqrt(2), []).rep == DMP([], QQ)
+    assert AlgebraicNumber(sqrt(2), (0, 0)).rep == DMP([], QQ)
```

Comment:
Well was `q = AlgebraicNumber(sqrt(2), ())` working before or not?  Seems to me that would have given the same as `q = AlgebraicNumber(sqrt(2), (1,0))` but it should give the zero element.  Now it gives the zero element.  (have I got that right?)

Hence I would like a test for `AlgebraicNumber(sqrt(2), ())` which before did something different from `AlgebraicNumber(sqrt(2), [])`.  But now, happily, both do the same thing.

Please correct me if I've got some of this wrong.